makefile.example

```diff
@@ -0,0 +1,104 @@
+# migrations placed under this directory
+# do not edit this section
+MIGRATION_DIR=./migrations
+
+# edit this to match your env dev|prod|test etc
+ENV=dev
+
+# ==========================================================
+# edit this fields to match your database enviroment
+# must match with app database
+# user name of database
+POSTGRES_USER=edunota
+
+# pasword of database
+POSTGRES_PASSWORD=edunota
+
+# database to apply migrations
+POSTGRES_DB=edunota
+
+# port of database
+POSTGRES_PORT=5432
+
+# hostname of database
+POSTGRES_HOST=localhost
+
+
+# ============================================
+# used by goose, do not edit these fields																								    
+export GOOSE_DBSTRING=user=$(POSTGRES_USER) password=$(POSTGRES_PASSWORD) host=$(POSTGRES_HOST) port=$(POSTGRES_PORT) dbname=$(POSTGRES_DB)
+export GOOSE_DRIVER=postgres
+# ============================================
+
+# ==========================================================
+# commands
+# spin up database and other services on docker compose
+spinup: 
+	bash -c "sudo docker compose --project-directory=infrastructure/$(ENV) up -d"
```

Comment:
I am windows user . sudo is not a windows comment

Reply:
This is a problem caused by windows, better switch a unix environment. or create your own makefile based on example